base/loading.jl

```diff
@@ -582,16 +583,17 @@ function create_expr_cache(input::String, output::String, concrete_deps::typeof(
                   empty!(Base._concrete_dependencies)
                   append!(Base._concrete_dependencies, $concrete_deps)
                   Base._track_dependencies[] = true
-                  end)
+                  end\0
```

Comment:
While effectively the same, might be clearer to splice in `repr(...)` explicitly?